src/Okta.Sdk.IntegrationTests/DisableTestParallelization.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
question: Do you have to create a new class for this setting to work? I wonder if we can configure this via xunit json or something like that?

Reply:
The information I read implied that xunit is scanning assemblies to determine appropriate behavior.  This setting is technically not in a class it's sitting in the global/unnamed namespace.

Reply:
Something like this would typically be placed in the AssemblyInfo.cs file, but because dotnet core will generate that file depending on compile time settings, putting it in it's own file seemed logical to me.

Reply:
It seems you can still configure this via csproject or xunit.runner.json. Have you tried this? See post #4 here: https://stackoverflow.com/questions/1408175/execute-unit-tests-serially-rather-than-in-parallel. If configuring via csproject or xunit.runner.json is possible, I think we should go that way. I feel it is more intuitive to have this type of configuration in some sort of configuration file.

Reply:
Certainly something we can try.  I'd like to reserve that for a future experiment considering the challenges faced getting this build green.